addons/pos_restaurant/static/src/overrides/models/pos_store.js

```diff
@@ -349,7 +349,7 @@ patch(PosStore.prototype, {
                 }
             }
             this.set_order(destinationOrder);
-            this.addPendingOrder([destinationOrder.id]);
+            this.addPendingOrder([destinationOrder?.id]);
```

Comment:
```suggestion
            if (destinationOrder?.id) {
                this.addPendingOrder([destinationOrder?.id]);
            }
```

I don't think it's a good idea to add undefined value in the pending order queue.

Reply:
@caburj Updated!